### PR TITLE
feat: add automated release to npm

### PR DESCRIPTION
**Pull request by BalbinaK**
Updates based on our [Automated releases guide](https://github.com/warp-ds/release-playground/wiki/Automate-releases-with-semantic-release-and-Eik)

Note:
There shouldn't be many updates in this repo, so it should be enough to open PRs directly to `master` and skip the `alpha`/`next` pre-releases.

### CODE REVIEW

**Comment by BalbinaK** [comment]
[Publish preset-migrate to NPM](https://trello.com/c/wPL3B8yh/197-publish-preset-migrate-to-npm)

File: package.json [hunk 1]
```diff
@@ -1,14 +1,34 @@
 {
   "type": "module",
+  "version": "1.0.0",
+  "name": "@warp-ds/migrate",
```
Comment (magnuh) [inline package.json]: Perhaps we should call it preset-migrate, to avoid any confusions? 😄

Comment (BalbinaK) [reply package.json]: Good point! Fixed in [a810bb3](https://github.com/warp-ds/preset-migrate/pull/3/commits/a810bb3df8e973b51dbc33e6ec8b7db0de976230)

Comment (magnuh) [reply package.json]: 🙌 😄

**Review by magnuh — APPROVED** [review]

**Pull request merged**